src/mixxx.cpp

```diff
@@ -124,6 +125,13 @@ MixxxMainWindow::MixxxMainWindow(QApplication* pApp, const CmdlineArgs& args)
     // after an upgrade and make any needed changes.
     m_pUpgrader = new Upgrade;
     m_pConfig = m_pUpgrader->versionUpgrade(args.getSettingsPath());
+
+    registerConfigPath(m_pConfig->getSettingsPath() + "/mixxx_qsettings_temp.cfg");
+    UserSettings user_settings;
+    // I need to explicitely ask for a reference because QSettings doesn't
+    // support move in Qt4
+    auto& qsettings = user_settings.getQSettings();
```

Comment:
This is a proof of concept. No setting has been switched so far. It should only show how qsettings can be used to replace the old system and still be backwards compatible.